src/app/modules/navbar/navbar.component.ts

```diff
@@ -140,7 +140,10 @@ export class NavbarComponent implements OnInit {
     .subscribe((data: any) => {
         this.reviewType  = data.reviewType;
     }, err => {
-      this.errorService.showErrorModal(err);
+      const e = new Error(err);
```

Comment:
@Harshwardhan18 Shouldn't this only be in places where the consultationProfile query is called?

Reply:
@avikt bro here also the query is being called on getCurrentUser (). In 3 components the query is being called instead of passing in chikd components. And again it will result in error modal at top.